protocol/common/messaging_persistence.go

```diff
@@ -0,0 +1,152 @@
+package common
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/status-im/status-go/messaging"
+)
+
+const tableName = "wakuv2_keys"
```

Comment:
Same here, would be nice to migrate this table to something `waku2` agnostic?

Reply:
Making the code waku-agnostic is still ahead of me; it should be addressed in follow-ups.